src/StableRNGs.jl

```diff
@@ -132,5 +139,20 @@ for T in Base.BitInteger_types
         SamplerRangeFast(r)
 end
 
+# https://github.com/JuliaRandom/StableRNGs.jl/issues/10
+Random.shuffle(r::StableRNG, a::AbstractArray) = Random.shuffle!(r, Base.copymutable(a))
```

Comment:
Is this one necessary?

Reply:
I'm not sure, but I would prefer to keep it, in case it becomes necessary. It doesn't seem harmful either.